orangecontrib/text/country_codes.py

```diff
@@ -319,13 +319,212 @@
+    'BN': ['Bandar Seri Begawani'],
+    'BO': ['La Paz', 'Sucre'],
+    'BR': ['Brasília', 'Brasilia', 'Sao Paulo', 'Sao Paulo', 'Rio de Janeiro'],
+    'BS': ['Nassau'],
+    'BT': ['Thimphu'],
+    'BW': ['Gaborone'],
+    'BZ': ['Belmopan'],
+    'CA': ['Ottawa', 'Vancouver', 'Calgary', 'Edmonton', 'Winnipeg', 'Toronto', 'Montreal', 'Québec', 'Quebec'],
+    'CD': ['Kinshasa'],
+    'CF': ['Bangui'],
+    'CG': ['Brazzaville'],
+    'CI': ['Yamoussoukro', 'Abidjan'],
+    'CL': ['Santiago'],
+    'CM': ['Yaoundé', 'Yaounde'],
+    'CN': ['Beijing'],
+    'CO': ['Bogotá', 'Bogota'],
+    'CR': ['San José', 'San Jose'],
+    'CU': ['Havana'],
+    'DJ': ['Djibouti'],
+    'DO': ['Santo Domingo'],
+    'EC': ['Quito'],
+    'EH': ['Laayoune'],
+    'ER': ['Asmara'],
+    'ET': ['Addis Ababa'],
+    'FJ': ['Suva'],
+    'FK': ['Stanley'],
+    'GA': ['Libreville'],
+    'GH': ['Accra'],
+    'GL': ['Nuuk'],
+    'GM': ['Banjul'],
+    'GN': ['Conakry'],
+    'GQ': ['Malabo'],
+    'GT': ['Guatemala City'],
+    'GW': ['Bissau'],
+    'GY': ['Georgetown'],
+    'HN': ['Tegucigalpa'],
+    'HT': ['Port-au-Prince', 'Port au Prince'],
+    'ID': ['Jakarta'],
+    'IN': ['New Delhi', 'Mumbai', 'Bombay', 'Bengaluru', 'Hyderabad'],
+    'IR': ['Tehran'],
+    'JM': ['Kingston'],
+    'JP': ['Tokyo', 'Osaka', 'Hiroshima', 'Fukuoka', 'Sapporo'],
+    'KE': ['Nairobi'],
+    'KG': ['Bishkek'],
+    'KH': ['Phnom Penh'],
+    'KP': ['Pyongyang'],
+    'KR': ['Seoul'],
+    'KW': ['Kuwait City'],
+    'KZ': ['Astana'],
+    'LA': ['Vientiane'],
+    'LK': ['Sri Jayawardenepura Kotte', 'Colombo'],
+    'LR': ['Monrovia'],
+    'LS': ['Maseru'],
+    'MG': ['Antananarivo'],
+    'ML': ['Bamako'],
+    'MM': ['Naypyidaw'],
+    'MN': ['Ulaanbaatar', 'Ulan Bator'],
+    'MR': ['Nouakchott'],
+    'MW': ['Lilongwe'],
+    'MX': ['Mexico City'],
+    'MY': ['Kuala Lumpur', 'Putrajaya'],
+    'MZ': ['Maputo'],
+    'NA': ['Windhoek'],
+    'NC': ['Nouméa', 'Noumea'],
+    'NE': ['Niamey'],
+    'NG': ['Abuja'],
+    'NI': ['Managua'],
+    'NP': ['Kathmandu'],
+    'NZ': ['Wellington', 'Auckland', 'Christchurch', 'Queenstown'],
+    'OM': ['Muscat'],
+    'PA': ['Panama City'],
+    'PE': ['Lima'],
+    'PG': ['Port Moresby'],
+    'PH': ['Manila'],
+    'PK': ['Islamabad'],
+    'PR': ['San Juan'],
+    'PY': ['Asunción', 'Asuncion'],
+    'QA': ['Doha'],
+    'RW': ['Kigali'],
+    'SB': ['Honiara'],
+    'SD': ['Khartoum'],
+    'SL': ['Freetown'],
+    'SN': ['Dakar'],
+    'SO': ['Mogadishu'],
+    'SR': ['Paramaribo'],
+    'SS': ['Juba'],
+    'SV': ['San Salvador'],
+    'SZ': ['Mbabane', 'Lobamba'],
+    'TD': ['N\'Djamena', 'NDjamena', 'N Djamena'],
+    'TG': ['Lomé', 'Lome'],
+    'TH': ['Bangkok'],
+    'TJ': ['Dushanbe'],
+    'TL': ['Dili'],
+    'TM': ['Ashgabat'],
+    'TT': ['Port of Spain'],
+    'TW': ['Taipei'],
+    'TZ': ['Dodoma'],
+    'UG': ['Kampala'],
+    'US': ['Washington, D.C.', 'Washington D.C.'],
```

Comment:
For inclusion of _capitals and larger cities_, this is fairly incomplete. :stuck_out_tongue: